src/data/data.json

```diff
@@ -1,7 +1,7 @@
 {
     "classifications": [
         { 
-            "index": 1,
+            "index": 0,
```

Comment:
you don't need to add index, because every element of an array has already a index, but it is ok for now

Reply:
yessss 👍